lib/rack/error_handler.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'constants'
+
+module Rack
+  class ErrorHandler
+    def initialize(app)
+      @app = app
+    end
+
+    def log_error(env, error)
+      if error_stream = env[RACK_ERRORS]
```

Comment:
I personally think it's good to be defensive here.

An example of where this could go wrong: If a request is missing `rack.errors`, and the `Rack::Lint` raises an exception, the error handler will also blow up.

I don't really have a strong opinion either way, so assuming you don't care about this case, I'll change it.

Reply:
You make a good point regarding Lint.  I'm not sure anyone would use Lint and ErrorHandler together.  Lint is a development (possibly test) tool, while ErrorHandler is more for production (basic error messages, error details not visible to the user).  However, I don't object to exiting early if there is no error stream.